public/js/admin.js

```diff
@@ -581,7 +581,7 @@ var deleteQuestion = function(qid) {
 
 var editQuestion = function(qid) {
     $.ajax({
-        type: 'POST',
+        type: 'GET',
```

Comment:
good call. the function name is missleading lol. GET is the correct type though.